config/_default/config.yaml

```diff
@@ -1,4 +1,4 @@
-baseURL: 'https://innersourcecommons.org/'
+baseURL: 'https://staging.iscf.site/'
```

Comment:
I presume we need to put this back to this value?

```
baseURL: 'https://innersourcecommons.org/'
```

Reply:
indeed...!! Thanks!

Reply:
Needs to go back to https://innersourcecommons.org/'